corehq/apps/app_manager/views/releases.py

```diff
@@ -290,12 +299,13 @@ def save_copy(request, domain, app_id):
             timer = datadog_bucket_timer('commcare.app_build.new_release', tags=[],
                                          timing_buckets=(1, 10, 30, 60, 120, 240))
             with timer:
-                copy = app.make_build(
-                    comment=comment,
-                    user_id=user_id,
-                )
-                copy.save(increment_version=False)
+                result = make_app_build(app, comment, user_id)
+                if isinstance(result, HttpResponse):
```

Comment:
I think you can remove this `if` statement now?

Reply:
Sorry, yes.